cristin-commons/src/main/java/no/unit/nva/HttpClientProvider.java

```diff
@@ -0,0 +1,17 @@
+package no.unit.nva;
+
+import java.net.http.HttpClient;
+import java.net.http.HttpClient.Version;
+import java.time.Duration;
+
+public class HttpClientProvider {
+
+    public static HttpClient defaultHttpClient() {
+        return HttpClient.newBuilder()
+                   .followRedirects(HttpClient.Redirect.ALWAYS)
+                   .version(Version.HTTP_1_1)
+                   .connectTimeout(Duration.ofSeconds(30))
```

Comment:
Occupying a server thread in Cristin for 30 seconds is a long time. Does it even exceed our lambda timeout? What would happen if we decreased this to e.g. 10 seconds from your experience?

Reply:
Should i change it to 10, maybe 15 seconds before merge or keep as is?

Reply:
I dont think it will be a problem to decrease to 15, but under high load in cristin i have experienced up to 1 minute response times, which gives us a gateway timeout from the load balancer